backend/tilavarauspalvelu/enums.py

```diff
@@ -92,11 +92,12 @@ class OrderStatus(models.TextChoices):
     EXPIRED = "EXPIRED", pgettext_lazy("OrderStatus", "Expired")
     CANCELLED = "CANCELLED", pgettext_lazy("OrderStatus", "Cancelled")
     PAID = "PAID", pgettext_lazy("OrderStatus", "Paid")
+    PAID_BY_INVOICE = "PAID_BY_INVOICE", pgettext_lazy("OrderStatus", "Paid by invoice")
```

Comment:
`PAID_BY_INVOICE` needs to be added to `OrderStatusWithFree` also

Reply:
Fixed and added regression test